mvnw

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
This should stay `!#/bin/sh` - it's not bash. And furthermore, this file is used asis from https://maven.apache.org/wrapper/maven-wrapper-plugin/index.html - if you update the wrapper using `wrapper:wrapper` this change would anyway be undone.